x/gov/spec/02_state.md

```diff
@@ -11,26 +11,14 @@ be one active parameter set at any given time. If governance wants to change a
 parameter set, either to modify a value or add/remove a parameter field, a new
 parameter set has to be created and the previous one rendered inactive.
 
-```go
-type DepositParams struct {
-  MinDeposit        sdk.Coins  //  Minimum deposit for a proposal to enter voting period.
-  MaxDepositPeriod  time.Time  //  Maximum period for Atom holders to deposit on a proposal. Initial value: 2 months
-}
-```
+### DepositParams
++++ https://github.com/cosmos/cosmos-sdk/blob/v0.40.0-rc7/proto/cosmos/gov/v1beta1/gov.proto#L127-L145
 
-```go
-type VotingParams struct {
-  VotingPeriod      time.Time  //  Length of the voting period. Initial value: 2 weeks
-}
-```
+### VotingParams
++++ https://github.com/cosmos/cosmos-sdk/blob/atheesh/gov-basic-cleanup/proto/cosmos/gov/v1beta1/gov.proto#L147-L156
```

Comment:
If you keep this branch, the link will be dead once the branch is deleted.

```suggestion
+++ https://github.com/cosmos/cosmos-sdk/blob/v0.40.0/proto/cosmos/gov/v1beta1/gov.proto#L147-L156
```

Reply:
My bad, thanks!